package motor_goal_position.msgs;

message MotorGoalPosition
{
    required double position = 1;
    required double maxVel = 2;
    required string name = 3;
}